app/lib/actions/pulp3/abstract_async_task.rb

```diff
@@ -82,7 +82,14 @@ def cancel!
       end
 
       def cancel
-        fail "Cancelling not supported"
+        output[:pulp_tasks].each do |pulp_task|
+          data = PulpcoreClient::Task.new
+          pulp3_api.tasks_api.tasks_cancel(pulp_task['_href'], data)
```

Comment:
why did you have to pass in a 'new' task object?  was it related to https://pulp.plan.io/issues/4883 ?